src/test-utils/src/evaluateGraphQLResolver.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+/**
+ * Use this function to evaluate resolvers in test files. Usage:
+ *
+ * ```js
+ * const fields = Location.getFields();
+ * expect(
+ *   evaluateGraphQLResolver(fields.countryFlagURL, {
+ *     country: ' ... ', // test value
+ *   }),
+ * ).toBe(' ... ');
+ * ```
+ */
+export default function evaluateGraphQLResolver(
+  field: { [key: string]: any, name: string, ... },
+  testValue: { [key: string]: any, ... },
+  argsValue?: { [key: string]: any, ... },
+  contextValue?: { [key: string]: any, ... },
+): any {
+  const resolveFn =
+    field.resolve ||
+    function resolveMock(): string {
+      return testValue[field.name];
```

Comment:
Why do we need this fallback value, shouldn't all fields have a resolve function?

Reply:
🤔  Oh, I guess it can be defined without one, and this is the default function 👍 

rubber-🦆-ing out loud here 😂

Reply:
But does this make sense then? Why would you call `evaluateGraphQLResolver` for no resolver?

Reply:
I mean, you define a field like this: 

```
format: {
      type: GraphQLString,
      description: 'Format string of the currency',
    },
```

id doesn't have a resolver, but by default it returns parent.format

Reply:
Because it's confusing. In the test you get `undefined` but in real usage you get some value.

Reply:
It is not only confusing, it is wrong if it works differently in the test than in the real usage

Reply:
> it doesn't have a resolver, but by default it returns parent.format

@tbergq That's however changeable behavior, see `fieldResolver` in the Apollo server for example. I am not convinced it's a good strategy to simply hardcode it like this.

Reply:
You mean resolvers map in Apollo server?

Reply:
<img width="782" alt="Screenshot 2020-04-08 at 16 05 38" src="https://user-images.githubusercontent.com/7705041/78793541-d56c3700-79b2-11ea-938a-dab8f250fc62.png">

@mrtnzlml you mean ☝️

Reply:
No, I mean `fieldResolver`: https://github.com/apollographql/apollo-server/blob/377be257b31d85a14dd2b001496194ed04ac2e03/packages/apollo-server-core/src/graphqlOptions.ts#L33

The same field exists in GraphQL.js for example. It would be at least nice to reuse the same logic if we already decided differently. Here it is (can be imported): https://github.com/graphql/graphql-js/blob/278bde0a5cd71008452b555065f19dcd1160270a/src/execution/execute.js#L1181-L1199

Reply:
Aha, nice, Yeah, I agree, lets use that one instead

Reply:
OK, let's move here please https://github.com/adeira/universe/pull/588